### PR TITLE
PLAT-636: Add schedule to the input which allows the use of service_type: cronjob

### PR DESCRIPTION
**Pull request by bjarneo**
Temp use the dev branch for the generate manifest

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -67,7 +71,7 @@ runs:
       id: manifest
       run: echo "::set-output name=filename::${{ inputs.instance }}-${{ inputs.app_name }}.k8s.manifest.yaml"
 
-    - uses: dignio/generate-manifest@v3
+    - uses: dignio/generate-manifest@PLAT-636/create-cronjob-generation
```
Comment (weel) [inline action.yaml:74]: Did you want to merge this back to generate-manifest first?

Comment (bjarneo) [reply action.yaml:74]: Yes, absolutely :)

Comment (bjarneo) [reply action.yaml:74]: First we have to review this PR https://github.com/dignio/generate-manifest/pull/13.

**Review by weel — APPROVED** [review]
LGTM 🌟

**Pull request merged**